### ISSUE TITLE
[Bug]: There is an incorrect error message for multi-cn self-increment in launch mode

### ISSUE THREAD

**Issue by nnsgmsone** [matrixorigin/matrixone#10204]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

Data truncation: data out of range: data type int unsigned, value 0

### Expected Behavior

Data truncation: data out of range: data type int unsigned, value 4294967296

### Steps to Reproduce

```Markdown
drop table t1;
create table t1(a int unsigned auto_increment);
insert into t1 values(null), (3), (null), (6), (null), (4294967295);
select * from t1;
insert into t1 values();
```

### Additional information

Multi-CN e2e BVT Compatibility Test on Linux/x64(LAUNCH, PUSH

**Comment by zhangxu19830126** [comment]
fixed by #11175

**Comment by matrix-meow** [comment]
Hello @nnsgmsone. The bug issue in the BVT test code has not been removed,issues automatically open.
